Resources/Prototypes/Entities/Mobs/Player/silicon.yml

```
@@ -637,3 +640,61 @@
+      - id: XrayCannon
+  - type: UserInterface
+    interfaces:
+    - key: enum.StrippingUiKey.Key
+      type: StrippableBoundUserInterface
+  - type: GhostTakeoverAvailable
+    makeSentient: true
+    name: Onestar Mecha
+    description: You are an experimental mecha created by who-knows-what, all you know is that you have weapons and you detect fleshy moving targets nearby...
+    rules: Use your weapons to cause havok. You are an antagonist.
+  - type: MovementSpeedModifier
+    baseWalkSpeed : 3
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)